test/hotspot/jtreg/ProblemList.txt

```diff
@@ -83,14 +83,9 @@ gc/stress/gclocker/TestGCLockerWithG1.java 8180622 generic-all
 gc/stress/TestJNIBlockFullGC/TestJNIBlockFullGC.java 8192647 generic-all
 gc/stress/TestStressG1Humongous.java 8286554 windows-x64
 
-gc/shenandoah/TestDynamicSoftMaxHeapSize.java#generational 8306333 generic-all
-gc/shenandoah/oom/TestThreadFailure.java 8306335 generic-all
-gc/shenandoah/oom/TestClassLoaderLeak.java 8306336 generic-all
-gc/shenandoah/mxbeans/TestChurnNotifications.java#generational 8306337 generic-all
-gc/stress/gcold/TestGCOldWithShenandoah.java#generational 8306339 generic-all
-gc/stress/systemgc/TestSystemGCWithShenandoah.java#generational 8306340 generic-all
-gc/stress/gclocker/TestGCLockerWithShenandoah.java#generational 8306341 generic-all
-gc/TestAllocHumongousFragment.java#generational 8306342 generic-all
```

Comment:
Since `TestAllocHumongousFragment` and `TestClassLoaderLeak` still fail in [some configs](https://github.com/earthling-amzn/shenandoah/actions/runs/4874732691), I'd leave them in the problem list until they are fixed, perhaps by reproducing on those platforms. At least one of the flagged failure modes for TestClassLoaderLeak (the failing assert) is likely fixed in Kelvin's upcoming PR, so we can revisit these again following that PR as well.